src/docfx/lib/process/ProcessUtility.cs

```diff
@@ -5,14 +5,20 @@
 using System.ComponentModel;
 using System.Diagnostics;
 using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
 
 namespace Microsoft.Docs.Build;
 
-/// <summary>
-/// Provide process utility
-/// </summary>
 internal static class ProcessUtility
 {
+    private static readonly JsonSerializerOptions s_jsonOptions = new()
+    {
+        AllowTrailingCommas = true,
+        Converters = { new JsonStringEnumConverter() },
+        PropertyNamingPolicy = new JsonSnakeCaseNamingPolicy(),
```

Comment:
Not sure if we will have all the models with snake case naming.

Reply:
For the consumer of this class, yes. But we use both snake case and camel case, so at some stage later we'll probably need an enum to specify the naming convention.